ojd_daps_skills/utils/bert_vectorizer.py

```diff
@@ -13,12 +13,10 @@ class BertVectorizer:
     def __init__(
         self,
         bert_model_name="sentence-transformers/all-MiniLM-L6-v2",
-        multi_process=True,
```

Comment:
oh interesting - why get rid of multi_process?

Reply:
it doesn't work on batch (I believe it's a system resources problem), and you can't set the bool to false from the ExtractSkills class (i think)

Reply:
interesting - this one might be worth adding as a `ExtractSkills` argument then